content/flux/v0.x/query-data/sql/percona.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Query Percona
+list_title: Percona
+description: >
+  Use [`sql.from()`](/flux/v0.x/stdlib/sql/from/) with the `mysql` driver to query Percona.
+menu:
+  flux_0_x:
+    name: Percona
+    parent: SQL databases
+weight: 101
+related:
+  - /flux/v0.x/stdlib/sql/from/
+list_code_example: |
+  ```js
+  import "sql"
+  
+  sql.from(
+    driverName: "mysql",
+    dataSourceName: "user:password@tcp(localhost:3306)/db",
+    query: "SELECT * FROM example_table"
+  )
+  ```
+---
+
+To query [Percona](https://www.percona.com/) with Flux, import the [`sql` package](/flux/v0.x/stdlib/sql/)
+and use the [`sql.from()` function](/flux/v0.x/stdlib/sql/from/) with the `mysql` driver.
+Provide the following parameters:
+
+- **driverName**: mysql
+- **dataSourceName**: [Percona data source name (DSN)](#data-source-name)
+  _(also known as **connection string**)_
+- **query**: SQL query to execute
+
+```js
+import "sql"
+
+sql.from(
+  driverName: "mysql",
+  dataSourceName: "user:password@tcp(localhost:3306)/db",
+  query: "SELECT * FROM example_table"
+)
+```
+
+##### On this page
+
+- [Data source name](#data-source-name)
+- [Data types](#data-types)
+- [Results structure](#results-structure)
+- [Store sensitive credentials as secrets](#store-sensitive-credentials-as-secrets)
+
+## Data source name
+The `mysql` driver uses the following data source name (DSN) syntax:
```

Comment:
Percona?

Reply:
Percona is just a fork of MySQL and uses the same driver.